frontend/src/components/common/Sidebar.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <nav class="p-sidenav">
+        <a
+            class="p-sidenav-logo"
+            title="Forest Access Management"
+            href="https://www2.gov.bc.ca"
+        >
+            <img
+                src="@/assets/images/17_gov3_bc_logo_transparent.svg"
+                alt="B.C. Government Logo"
+            />
+        </a>
+        <div class="content">
+            <ul>
+                <div v-for="item in props.data">
+                    <li class="header">{{ item.name }}</li>
+                    <ul>
+                        <li
+                            v-for="child in item.items"
+                            class="child"
+                            :class="{
+                                'p-sidenav-selected':
+                                    $router.currentRoute.value.path ==
+                                    child.link,
+                                'p-sidenav-disabled': child.disabled,
+                            }"
+                            @click="router.push(child.link)"
+                        >
+                            <Icon
+                                class="p-sidenav-color-icon p-sidenav-padding-icon"
+                                small
+                                :icon="child.icon.toString()"
+                            ></Icon>
+                            {{ child.name }}
+                        </li>
+                    </ul>
+                </div>
+            </ul>
+        </div>
+
+        <div class="support-section p-sidenav-disabled">
+            <ul>
+                <li class="header">Support</li>
+                <ul>
+                    <li
+                        class="child"
+                        click="mailto:SIBIFSAF@Victoria1.gov.bc.ca"
+                    >
+                        <Icon
+                            icon="HelpIcon"
+                            small
+                            class="p-sidenav-padding-icon"
+                        />
+                        Need help?
+                    </li>
+                </ul>
+            </ul>
+        </div>
+    </nav>
+</template>
+
+<script setup lang="ts">
+import router from '@/router';
+import type { PropType } from 'vue';
+import type { RouteLocationRaw } from 'vue-router';
+
+interface ISideBarData {
+    name: string;
+    items: [ISideBarItem];
+}
+
+interface ISideBarItem {
+    name: string;
+    icon: string;
+    link: RouteLocationRaw;
+    disabled: boolean;
+}
```

Comment:
Since you are renaming SideBar to SideNav, please don't forget to rename ISideBarData and ISideBarItem to be consistent.